pom.xml

```diff
@@ -70,29 +70,11 @@
     <dependency.org.slf4j.slf4j-api.version>1.7.7</dependency.org.slf4j.slf4j-api.version>
 
     <!-- Test dependency properties-->
-    <dependency.junit.junit.version>4.11</dependency.junit.junit.version>
     <dependency.org.mockito.mockito-all.version>1.9.5</dependency.org.mockito.mockito-all.version>
     <dependency.org.slf4j.slf4j-log4j12.version>1.7.7</dependency.org.slf4j.slf4j-log4j12.version>
 
   </properties>
 
-  <build>
-    <plugins>
-      <!-- Override here to allow "mvn checkstyle:checkstyle" to work against pentaho standards -->
-      <plugin>
```

Comment:
we added this here to allow us to run checkstyles without having to build the entire site. does this exist in the parent pom? if not, can you add it?